packages/client/hmi-client/src/workflow/tera-workflow-node.vue

```diff
@@ -1,26 +1,12 @@
 <template>
 	<main :style="nodeStyle" ref="workflowNode">
-		<header :class="{ 'active-node': isActive }">
-			<h5 class="truncate">{{ node.displayName }}</h5>
-			<span>
-				<Button
-					icon="pi pi-search-plus"
-					class="p-button-icon-only p-button-text p-button-rounded"
-					@click="showNodeDrilldown"
-				/>
-				<Button
-					icon="pi pi-bolt"
-					class="p-button-icon-only p-button-text p-button-rounded"
-					@click="openDrilldown"
-				/>
-				<!-- 3-dot options menu -->
-				<Button
-					icon="pi pi-ellipsis-v"
-					class="p-button-icon-only p-button-text p-button-rounded"
-					@click="toggleNodeMenu"
-				/>
-				<Menu ref="nodeMenu" :model="nodeMenuItems" :popup="true" />
-			</span>
+		<header>
+			<tera-operator-header
+				:name="node.displayName"
+				:status="node.status"
+				@open-in-new-window="openDrilldown"
+				@remove-node="emit('remove-node', props.node.id)"
```

Comment:
We need to simplify those events name.

Reply:
Please provide more detail, rationale, suggestion or whatnot to help facilitate the discussion. It is hard to know what is the concern or what are the desired effects with these short declarations.

Reply:
`open-in-new-window` here is not used to open in a new window, but the drilldown.
`remove-node` is too verbose.

We can simplify the events to be:
- `open-drilldown`
- `open-in-new-window`
- `remove-operator`
- `bring-to-front`
- `send-to-back`